apps/champions/test/support/test_utils.ex

```diff
@@ -13,7 +13,7 @@ defmodule Champions.TestUtils do
         class: "Assassin",
         base_attack: 100,
         base_health: 100,
-        base_defense: 100,
+        base_defense: 0,
```

Comment:
why changing this?

Reply:
It makes more sense for default values to be as "useless" as possible (skills that never get to be executed, for example). The neutral value for defense is 0 since it just leaves the damage received unmodified, so it's a more natural fit

Reply:
nice